addons/sourcemod/scripting/gokz-localranks/db/update_ranked_map_pool.sp

```diff
@@ -7,7 +7,18 @@
 
 void DB_UpdateRankedMapPool(int client)
 {
-	Handle file = OpenFile(LR_CFG_MAP_POOL, "r");
+	if (g_DBType != DatabaseType_SQLite && g_DBType != DatabaseType_MySQL)
+	{
+		LogError("Unsupported database type, cannot update map pool");
+		if (IsValidClient(client))
+		{
+			// TODO Translation phrases?
```

Comment:
Yes indeed, please put that in a translation file.

Reply:
Should this condition just be removed? Perhaps we should assert for unsupported database drivers elsewhere (i.e. when we connect). Otherwise this seems a little of out place when it's the only database related function doing it.

Reply:
Yeah if we don't do that anywhere else we might as well remove it.